inc/html.class.php

```diff
@@ -1788,6 +1788,19 @@ static function simpleHeader($title, $links = []) {
 
       // Preferences + logout link
       echo "<div id='c_preference'>";
+      if (Session::getLoginUserID()) {
```

Comment:
I do not understand why you add a new logout link here

Reply:
This simple header is shown one the password update page which is displayed when a user logs in with an expired password. As this is the only page that user can access, we have to add this logout link to give hi ability to close its session.

Reply:
Ok :)